spec/models/edition_spec.rb

```diff
@@ -207,4 +207,40 @@
       end
     end
   end
+
+  describe "#assign_access_limit" do
+    it "can set an access limit" do
+      edition = build(:edition)
+      user = build(:user)
+
+      expect { edition.assign_access_limit(limit_type: :all_organisations, user: user) }
```

Comment:
Minor, but would it be OK to lose this particular expectation? It looks like it's better covered by the subsequent ones, and would make the setup/run/test phases clearer.

Reply:
I've removed this as I agree about the subsequent assertions.